task2/config/webpack.prod.js

```diff
@@ -0,0 +1,73 @@
+const util = require('./util');
+const {
+    merge
+} = require('webpack-merge');
+const common = require('./webpack.common.js');
+const HTMLWebpackPlugin = require('html-webpack-plugin');
+const MiniCssExtractPlugin = require('mini-css-extract-plugin');
+const CssMinimizerPlugin = require('css-minimizer-webpack-plugin');
+const TerserPlugin = require("terser-webpack-plugin");
+const {
+    BundleAnalyzerPlugin
+} = require('webpack-bundle-analyzer');
+
+module.exports = merge(common, {
+    mode: 'production',
+    output: {
+        filename: `[name].[contenthash].js`,
+        path: util.build,
+    },
+    plugins: [
+        new HTMLWebpackPlugin({
+            template: `${util.public}/index.html`,
+            title: `Webpack config`,
+            favicon: `${util.public}/assets/favicon.ico`,
+            minify: {
+                collapseWhitespace: true
+            }
+        }),
+        // Extracts CSS into separate files
+        // Note: style-loader is for development, MiniCssExtractPlugin is for production
```

Comment:
But why _style-loader is for development, MiniCssExtractPlugin is for production_?

Reply:
Ohhh.. We can leave it empty. Understood. I waste 3 days to understand, why my css contains invalid paths. Thanks!
You're right. We can change it to MiniCssExtractPlugin